monero/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: monero_server_1
+      APP_PORT: 9976
+      PROXY_AUTH_ADD: "true"
+
+  server:
+    image: deverickapollo/umbrel-monero:v1.0.0@sha256:a5ca2854b71de83a26a4176bf82f92609b0e67f54be5b7eb9386acffba87b33e
+    depends_on: [monerod]
+    restart: on-failure
+    expose:
+      - "9976"
+    volumes:
+      - ${APP_DATA_DIR}/data/app:/data # volume to persist advanced settings json
+      - ${APP_MONERO_DATA_DIR}:/monero/.monero # volume to persist bitmonero.conf
+    environment:
+      PORT: "9976"
+      MONERO_HOST: "${APP_MONERO_NODE_IP}"
+      MONERO_P2P_PORT: "${APP_MONERO_P2P_PORT}"
+      MONERO_RPC_PORT: "${APP_MONERO_RPC_PORT}"
+      MONERO_DEFAULT_NETWORK: "${MONERO_DEFAULT_NETWORK}"
+      MONERO_RPC_USER: "${APP_MONERO_RPC_USER}"
+      MONERO_RPC_PASSWORD: "${APP_MONERO_RPC_PASS}"
+      RPC_PORT: "${APP_MONERO_RPC_PORT}"
+      RPC_USER: "${APP_MONERO_RPC_USER}"
+      RPC_PASSWORD: "${APP_MONERO_RPC_PASS}"
```

Comment:
I think these three env vars are actually legacy code that we need to remove from our Bitcoin Node app, so you may be able to remove these without issue.

You can see that they are essentially duplicates of:

```
MONERO_RPC_PORT: "${APP_MONERO_RPC_PORT}"
MONERO_RPC_USER: "${APP_MONERO_RPC_USER}"
MONERO_RPC_PASSWORD: "${APP_MONERO_RPC_PASS}"
```